src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallFeatureTask.groovy

```diff
@@ -17,19 +17,22 @@ package net.wasdev.wlp.gradle.plugins.tasks
 
 import org.gradle.api.tasks.TaskAction
 
-class InstallFeatureTask extends AbstractTask {
+class InstallFeatureTask extends InstallLibertyTask {
 
     @TaskAction
     void installFeature() {
+        // install Liberty if not installed yet
+        install()
```

Comment:
@dshimo recently updated Liberty.groovy so that installFeature dependsOn installLiberty.  Do we still need this call to install()?  And if we don't, this file can still extend AbstractTask.